README.md

```diff
@@ -13,4 +13,4 @@ and use it 3 times inside the `App` instead of static markup.
 - Install Prettier Extention and use this [VSCode settings](https://mate-academy.github.io/fe-program/tools/vscode/settings.json) to enable format on save.
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_person/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://HunchakAndrii.github.io/react_person/) and add it to the PR description.
```

Comment:
Add the demo link to the PR description 
<img width="994" alt="image" src="https://github.com/user-attachments/assets/c6ce93cc-6b2a-4949-9035-72ab6d2a1355">

Reply:
Sorry